client/src/components/StreamField/scss/components/c-sf-add-button.scss

```diff
@@ -1,21 +1,25 @@
+$icon-size: theme('spacing.4');
+
 .c-sf-add-button {
```

Comment:
I’m not so happy this and the Draftail button still are separate even though they’re meant to look the same, but we have to revisit this anyway to ideally switch to a more reliable "circled outline plus" icon, so I’ve only done basic tweaks for now.